scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/controller/OutputController.java

```diff
@@ -306,7 +306,7 @@ public void downloadFullJobLogs(String sessionId, String jobId) {
 
     public void downloadFullTaskLogs(String sessionId, String jobId, String taskName) {
         String url = SchedulerConfig.get().getRestPublicUrlIfDefinedOrOverridden() + "/scheduler/jobs/" + jobId +
-                     "/tasks/" + taskName + "/result/log/full?sessionid=" + sessionId;
+                     "/tasks/" + taskName.replace("#", "%23") + "/result/log/full?sessionid=" + sessionId;
```

Comment:
it would be better to use a dedicated encodeUriComponent-like method if there is one already available in the scheduling-portal project

Reply:
I think you can use URL.encodePathSegment(segment)